src/Entity/Produto.php

```diff
@@ -42,12 +42,11 @@ class Produto implements JsonSerializable
 
     #[ORM\ManyToOne]
     #[Groups('produto')]
-    private Categoria $categoriaId;
+    private ?Categoria $categoria = null;
 
-    #[ORM\ManyToOne(inversedBy: 'produto_id')]
-    private ?Item $item = null;
 
-    public function __construct(){
+    public function __construct()
+    {
         $this->criadoEm = new \DateTimeImmutable('now', new \DateTimeZone('America/Sao_Paulo'));
```

Comment:
melhor configurar a zona no php da sua máquina... para ele a hora sempre vai estar certa!
mas legal saber que vocês aprenderam isso!

```suggestion
    {
        $this->criadoEm = new \DateTimeImmutable();
```